### PR TITLE
feat: added timestamps, a small margin at the bottom, and contributor notes

### PR DESCRIPTION
**Pull request by ConnorDoesDev**
me when 🗿

### CODE REVIEW

**Review by jackmthws — CHANGES_REQUESTED** [review]
how tf do you comment on commit content wtf

File: src/components/Settings.tsx [hunk 1]
```diff
@@ -25,8 +25,9 @@ export default ({ settings }: SettingsProps) => {
 
 	React.useEffect(() =>
 		() => {
-			if (settings.get('applicationId', null) && settings.get('name', null)) {
+			if (settings.get('enabled')) {
```
Comment (jackmthws) [inline src/components/Settings.tsx]: Unto this (reason below)

File: src/components/Settings.tsx [hunk 2]
```diff
@@ -35,6 +36,18 @@ export default ({ settings }: SettingsProps) => {
...
+					label="Enable Rich Presence"
+					subLabel="Enable or disable the Rich Presence"
+					trailing={(
+						<FormSwitch
+							value={settings.get('enabled')}
+							onValueChange={(value) => settings.set('enabled', value)}
+						/>
+					)}
+				/>
+                </FormSection>
```
Comment (jackmthws) [inline src/components/Settings.tsx]: Undo this too. Presence can be controlled by just turning the plugin on/off

**Review by jackmthws — CHANGES_REQUESTED** [review]
Other than these this should be good

**Comment by ConnorDoesDev** [comment]
> how tf do you comment on commit content wtf

🗿

**Comment by ConnorDoesDev** [comment]
done 🗿

**Review by jackmthws — APPROVED** [review]

**Pull request merged**